managed/services/management/checks.go

```diff
@@ -122,17 +122,23 @@ func (s *ChecksAPIService) GetFailedChecks(ctx context.Context, req *managementp
 
 	failedChecks := make([]*managementpb.CheckResult, 0, len(results))
 	for _, result := range results {
+		labels := make(map[string]string, len(result.Target.Labels)+len(result.Result.Labels)+4)
```

Comment:
What's this magic number 4 about?

Reply:
Leftover after refactoring. Before there were 4 extra labels for notifications that we send to AM.

Removed